File: cash.c
#include<stdio.h>
#include<cs50.h>

int main(void)
{
    int coins = 0; //Count # of coins
    float change = get_float("Change:"); //Get amount to change

    //float change; //Calculate remaining change

    coins = change / 0.25;
    change = change - coins*0.25;

    coins = change / 0.10;
    change = change - coins*0.10;

    coins = change / 0.05;
    change = change - coins*0.05;

    coins = change / 0.05;
    change = change - coins*0.01;

    printf("# of coins used: %i\n", coins);

    /*for (i = change; i > 0.25; i - 0.25)
    {
        count++;
    }
    for (i = total.change - count*0.25; i > 0.10; i - 0.10)
    {
        count++;
    }*/
}

/*2.56
take 0.25 , remaining > 0.25
take money divide per 0.25 , take integer

coins = integer
money = money - integer

take money divide per 0.1 , take integer*/

File: cs50/aula6/cash.py
from cs50 import get_float


def main():
    coins = 0
    while True:
        change = get_float("Change owed: ")
        if change > 0:
            break

    change = int(change * 100)
    for coin in [25, 10, 5, 1]:
        if change > 0:
            coins += change // coin
            # print(change//coin)
            change -= (change // coin) * coin
            # print(coins)
    print(coins)


main()
